app/Http/Controllers/Api/AssetsController.php

```diff
@@ -798,4 +799,67 @@ public function requestable(Request $request)
         $assets = $assets->skip($offset)->take($limit)->get();
         return (new AssetsTransformer)->transformRequestedAssets($assets, $total);
     }
+
+    /**
+     * Upload image for asset
+     * Supported formats are based on your image driver
+     *
+     * GD:      JPEG, PNG, GIF
+     * Imagick: JPEG, PNG, GIF, TIF, BMP, ICO, PSD
+     *
+     * @author [M. Skrip] [<marian.skripp@gmail.com>]
+     * @param int $asset_id
+     * @since [v4.7]
+     * @return JsonResponse
+     */
+    public function image(Request $request, $asset_id)
+    {
+        $this->authorize('image', Asset::class);
```

Comment:
We don't have a permission mask for image, do we?

Reply:
Removed